tests/client/output/test_types.py

```diff
@@ -44,7 +44,7 @@ def test_fieldspec_init():
             ]
         ),
     )
-    assert f.field_ref == "key_foo { bar baz }"
+    assert f.field_ref == "key_foo { bar baz baz }"
```

Comment:
Shouldn't the addition be `bbb`?

Reply:
That line is a test to check the field_ref, so 
{baz: fieldspec{field_ref:baz,alt_name:bbb}, bbb: fieldspec{field_ref:baz,alt_name:bbb}}
It looks awkward because it is in the form of , but it is correct that field_ref has two baz.

Reply:
Got it!